2019/WG-08-28.md

```diff
@@ -0,0 +1,62 @@
+![WebAssembly logo](/images/WebAssembly.png)
+
+## Agenda for the August 28 video call of WebAssembly's Working Group
+
+- **Where**: zoom.us
+- **When**: August 28, 2019 at 4pm-5pm UTC *( August 28, 2019 8am-9am PT )*
+- **Location**: *on calendar invite to registered attendees*
+- **Acting Chairs**:
+    - Derek Schuff
+    - Deepti Gandluri
+
+### Registration
+
+If you are a Working Group member no registration is required.
+
+If you are a Community Group member who would like to observe, please register
+here:
+https://goo.gl/forms/HD2kLCM0iSKk7AVl1
+
+## Logistics
+
+The meeting will be on a zoom.us video conference.
+Installation is required, see the calendar invite.
+
+## Agenda items
+
+1. PR transition
+
+## Agenda items for future meetings
+
+None.
+
+### Schedule constraints
+
+None.
+
+# Disposition Comments/PRs since CR fork 18 July
```

Comment:
Maybe the top-level header here should be "meeting notes" and "Disposition comments" could be a sub-heading?